lab1/Syntax.in.txt

```diff
@@ -0,0 +1,23 @@
+program = "begin" ";" decllist ";" cmpdstmt end ";"
+decllist = declaration | declaration ";" decllist
+declaration = "declare as " type ":" IDENTIFIER ";"
+type1 = "boolean" | "character" | "int" | “string”
+arraydecl = "list" "[" nr "]" "OF" type1
+type = type1|arraydecl
+cmpdstmt = "{" stmtlist "}"
+stmtlist = stmt | stmt ";" stmtlist
+stmt = simplstmt | structstmt
+simplstmt = assignstmt | iostmt
+assignstmt = IDENTIFIER "=" expression ";"
+expression = expression ("+" | "-") term | term
+term = term ("*" | "/") factor | factor
```

Comment:
maybe you will also need "%" or "**". You defined ** in your lexic, so you also should put it here. If you also need % you should also put it in your lexic.